apps/atomy-q/API/app/Http/Controllers/Api/V1/NormalizationController.php

```
@@ -186,6 +199,92 @@ public function sourceLines(Request $request, string $rfqId): JsonResponse
         ]);
     }
 
+    /**
+     * @param array<string, mixed>|null $providerProvenance
+     * @param array<string, mixed>|null $latestOverride
+     * @return array{rfq_line_item_id: string|null, quantity: string|null, uom: string|null, unit_price: string|null}|null
+     */
+    private function providerSuggestedValues(
+        NormalizationSourceLine $line,
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$line'. (undefined)

(UnusedFormalParameter)